tests/e2e/tests/content-manager/history.spec.ts

```diff
@@ -224,23 +224,25 @@ describeOnCondition(edition === 'EE')('History', () => {
       await findAndClose(page, 'Saved Document');
 
       // Delete one of the authors, leaving only Coach Beard
-      // await clickAndWait(page, page.getByRole('button', { name: 'Will Kitman' }));
-      // TODO: we need to renable this part when we implement the fullpage button
-      // await page.getByRole('button', { name: 'More actions' }).click();
-      // await page.getByRole('menuitem', { name: /delete entry/i }).click();
-      // await page.getByRole('button', { name: /confirm/i }).click();
-
-      // // Go to the history page
-      // await clickAndWait(page, page.getByRole('link', { name: 'Article' }));
-      // await clickAndWait(page, page.getByRole('gridcell', { name: 'Zava retires' }));
-      // await page.waitForURL(ARTICLE_EDIT_URL);
-      // await goToHistoryPage(page);
-      // await page.waitForURL(ARTICLE_HISTORY_URL);
-
-      // // Assert that the unknown relation alert is displayed
-      // await expect(page.getByRole('button', { name: 'Coach Beard' })).toBeVisible();
-      // await expect(page.getByText('Will Kitman')).not.toBeVisible();
-      // await expect(page.getByText(/missing relation/i)).toBeVisible();
+      // open the modal
```

Comment:
Real nitpick but can you keep comment styling consistent. Make sure your comments are specific and easy for people to understand what is happening. For example, what modal?

```suggestion
      // Open the relation modal
```